### ISSUE TITLE
Crown and Head Dupes

### ISSUE THREAD

**Issue by Mellohi138** [TeamChocoQuest/ChocolateQuestRepoured#280]
**Common sense Info**
To check a checkbox: Insert a lower case X between the square brackets, you need to remove the space though.
 - I play...
   - [ ] With a large modpack
   - [X] Only with CQR and it's dependencies
 - The issue occurs in...
   - [X] Singleplayer
   - [X] Multiplayer
 - [X] I have searched for this or a similar issue before reporting and it was either (1) not previously reported, or (2) previously fixed  and I'm having the same problem.
 - [X] I am using the latest version of the mod (all versions can be found on github under releases)
 - [X] I read through the FAQ and i could not find something helpful ([FAQ](https://wiki.cq-repoured.net/index.php?title=FAQ))
 - [X] I reproduced the bug without any other mod's except forge, cqr and it's dependencies
 - [ ] The game crashes because of this bug
 
 
 **Versions**
Chocolate Quest Repoured ('latest' IS NOT a valid version! Please use the version string that shows up in the mod overview of forge): 2.5.3B
Forge:  14.23.5.2855
Minecraft: 1.12.2

**Describe the bug**
It's a dupe bug with stacked heads and crown. I think it's caused with the crown attachment feature not knowing how to act when the item they're being used on is stacked.

**To Reproduce**

NOTE: I'm going to use 31 wither skulls in here as an example. But please note that any type of head with any type of number works.
1. Open a crafting GUI. 2x2 or 3x3. Any type.
2. Place 31 wither skulls with a crown. 
3. The output should be 31 wither skulls with crown. And when you take the crowned skulls out the crafting grid only wastes 1 skull.
4. When you try to take the crown out of the 31 skulls, you get 31 more normal skulls automatically per 1 crown take out.

**Expected behavior**
Basically both the skulls and crowns getting duped because of it.

**Screenshots**
https://cdn.discordapp.com/attachments/825840715631165501/842868237015318528/unknown.png
https://cdn.discordapp.com/attachments/825840715631165501/842868257358348310/unknown.png
https://cdn.discordapp.com/attachments/825840715631165501/842869717634646016/unknown.png
https://cdn.discordapp.com/attachments/825840715631165501/842871242087858226/unknown.png

**Other**
 - Link to Crash report file: -none-
 - Link to Log file: https://pastebin.com/nVihzr4X
 - Other mods (that are necessary to produce the bug):  -none-
 - Other mods (that are also installed): Geckolib - 3.0.0B
 - CQR config folder (link, only if you changed the configs): -not changed-
 - Other mod's configs (link): -none-

**Comment by DerToaster98** [comment]
I don't even know why it is attachable to skulls, that shouldn't be the case

**Comment by Mellohi138** [comment]
Probably because the player can wear them, just like helmets.

**Comment by DerToaster98** [comment]
ye, the code only checked if the item can be worn on the head and didn't care about stack sizes

**Comment by DerToaster98** [comment]
Fixed in 2.6.0B
